doc/docs/usage/tutorial-vehicle-signals.md

```diff
@@ -340,141 +340,3 @@ dependencies:
-  speed-consumer:
-    runtime: podman
-    agent: infotainment
-    dependencies:
-      databroker: ADD_COND_RUNNING
-    configs:
-      network: network
-      env: env_consumer # (7)!
-    runtimeConfig: | # (8)!
-      image: ghcr.io/eclipse-ankaios/speed-consumer:0.1.2
-      commandOptions:
-        - "--net={{network}}"
-        {{#each env}}
-        - "-e {{this.key}}={{this.value}}"
-        {{/each}}
-configs: # (9)!
-  network: host
-  env_provider:
-    - key: SPEED_PROVIDER_MODE
-      value: auto
-  env_consumer:
-    - key: KUKSA_DATA_BROKER_ADDR
-      value: "127.0.0.1"
-  agents:
-    name: agent_A
-```
-
-1. The agent name is templated and rendered with the configuration value that the 'agent' alias refers to, which is 'agent_A'.
-2. The configuration item 'agents' is assigned to the workload with alias 'agent'.
-3. The configuration item 'network' is assigned to the workload with alias 'network'.
-4. The runtimeConfig contains a template string accessing the assigned network configuration item. It is rendered with the configuration value that the 'network' alias refers to, which is 'host'.
-5. The configuration item 'env_provider' is assigned to the workload with alias 'env'.
-6. In addition to the templated string for the network, the runtimeConfig contains a templated loop to assign all environment variables that the 'env' alias refers to, which is 'SPEED_PROVIDER_MODE' with value 'auto'.
-7. The configuration item 'env_consumer' is assigned to the workload with alias 'env'.
-8. In addition to the templated string for the network, the runtimeConfig contains a templated loop to assign all environment variables that the 'env' alias refers to, which is 'KUKSA_DATA_BROKER_ADDR' with value '127.0.0.1'.
-9. The configuration items are defined as key-value pairs.
-
-Start the Ankaios cluster again, by executing the following command:
-
-```shell
-sudo systemctl start ank-server
-sudo systemctl start ank-agent
-```
-
-Start the `infotainment` agent, remembering to change the server URL if the agent is not running on the same host:
-
-```shell
-ank-agent -k --name infotainment --server-url http://127.0.0.1:25551
-```
-
-Verify again that all workloads are up and running.
-
-### Update configuration items
-
-Let's update the content of a configuration item with the `ank apply` command.
-
-Using `ank apply`:
-
-```yaml title="new-manifest.yaml"
-apiVersion: v0.1
-configs:
-  env_provider:
-    - key: SPEED_PROVIDER_MODE
-      value: webui
-```
-
-```shell
-ank -k apply new-manifest.yaml
-```
-
-Ankaios will update workloads that reference an updated configuration item.
-After running one of these commands, the `speed-provider` workload has been updated to run in the 'webui' mode.
-
-You can verify this by re-opening the web UI on <http://127.0.0.1:5000>.
-
-### List configuration items
-
-Let's list the configuration items present in current state with the `ank get configs` command.
-
-Using `ank -k get configs`, it should print:
```

Comment:
The `get configs` and the `delete configs` command seems not to be appearing in the user doc anymore. Is it planned to introduce it in the new tutorial somehow?

Reply:
A tutorial can only cover certain aspects for a specific use case. I think for fleet management somehow those commands do not fit. When using the CLI, the user will look at the help and then detect all available commands. The tutorial should just be an ice breaker.

Reply:
Ok, then I think we do not need add explicit documentation.